src/Microsoft.AspNetCore.Mvc.Testing/Internal/TestCulture.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Mvc.Testing.Internal
+{
+    public class TestCulture
+    {
+        public string Culture { get; set; }
```

Comment:
Should this just be `CultureInfo`? It would match what we do in Localization (https://github.com/aspnet/Localization/blob/dev/src/Microsoft.AspNetCore.Localization/RequestLocalizationOptions.cs#L92)